client/coral-plugin-commentbox/CommentBox.js

```diff
@@ -53,12 +71,12 @@ class CommentBox extends React.Component {
         // Execute postSubmit Hooks
         this.state.hooks.postSubmit.forEach(hook => hook(data));
 
+        notifyForNewCommentStatus(addNotification, postedComment.status);
+
         if (postedComment.status === 'REJECTED') {
-          addNotification('error', lang.t('comment-post-banned-word'));
-          !isReply && setCommentCountCache(commentCountCache);
+          !isReply && setCommentCountCache(assetId, commentCountCache);
         } else if (postedComment.status === 'PREMOD') {
-          addNotification('success', lang.t('comment-post-notif-premod'));
-          !isReply && setCommentCountCache(commentCountCache);
+          !isReply && setCommentCountCache(assetId, commentCountCache);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/14221600/25950519/6ba1eefa-3685-11e7-82dd-42fb47c0b188.png)
I have an error when posting a comment and premod is on.
Why was these two lines changed? Is it a merge error?